sequencer/src/state.rs

```diff
@@ -971,6 +959,12 @@ impl CheckedSub for FeeAmount {
     }
 }
 
+impl FeeAmount {
+    pub(crate) fn as_u64(&self) -> u64 {
+        self.0.as_u64()
```

Comment:
Yeah...the builder fee amount is currently a `u64` which leads to this awkwardness. It should be an associated type so we can define it in the sequencer as `U256`. But in the meantime I can try to rewrite this to fail validation if it doesn't fit instead of panicking

Reply:
Fixed in [390db12](https://github.com/EspressoSystems/espresso-sequencer/pull/1361/commits/390db12cd97faf824e32d1b54134859f62251ceb)